packages/manager/apps/dedicated/client/app/dedicated/server/install/image/components/config-drive/index.html

```diff
@@ -0,0 +1,149 @@
+<div class="dedicated-server-install-image-config-drive">
+    <h3 data-translate="dedicated_server_install_image_config_drive"></h3>
+
+    <p
+        data-translate="dedicated_server_install_image_config_drive_eplain"
+        data-translate-values="{ href: $ctrl.documentationLink }"
+    ></p>
+
+    <oui-field data-size="xl">
+        <oui-switch
+            data-model="$ctrl.model.configdrive.enabled"
+            data-id="activateConfigDrive"
+            data-name="activateConfigDrive"
+            data-disabled="$ctrl.loaders.launchInstall"
+        >
+        </oui-switch>
+        <label
```

Comment:
Why not use `label` field on `oui-field` ?

Reply:
Because the label of a `oui-field` is above the field. Here the label must be next to the field (`oui-switch`)